src/SDK/Client/Client.ts

```diff
@@ -61,6 +68,17 @@ export class Client extends EventEmitter {
 
         this.network = this.options.network ? this.options.network.toString() : 'testnet';
 
+        const mappedProtocolVersion = Client.networkToProtocolVersion.get(
+            this.network,
+        );
+
+        // use protocol version from options if set
+        // use mapped one otherwise
+        // fallback to one that set in dpp as the last option
+        this.driveProtocolVersion = this.options.driveProtocolVersion !== undefined
+          ? this.options.driveProtocolVersion
+          : (mappedProtocolVersion !== undefined ? mappedProtocolVersion : latestProtocolVersion);
```

Comment:
It seems Platform class is more suitable for this logic since Client is more general and responsible for both Core and Platform.